src/app/components/arlas-wui-root/arlas-wui-root.component.scss

```diff
@@ -545,4 +553,92 @@
       padding-bottom: 7px;
     }
   }
+
+  .extra-shortcuts {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-left: $sm-spacing;
+
+    .title {
+      cursor: pointer;
+      display: flex;
+      background-color: #fff;
+      justify-content: space-between;
+      align-items: center;
+      padding: 0 $sm-spacing;
+      // Due to box sizing on this element and not other filters, it is necessary to add the border width
+      height: calc($filter-height + 2 * $xs-border);
+      font-size: $default-font-size;
+      @include box-border();
+      background-color: $default-background-color;
+      // Necessary to maintain same alignement with extra shortcuts when open and closed
+      box-sizing: border-box;
+
+      .info {
+        display: flex;
+        justify-content: flex-end;
+        min-width: 50px;
+        padding-left: $sm-spacing;
+
+        .tablet {
+          @include tablet();
+        }
+
+        .chevron {
+          height: $icon-size;
+          width: $icon-size;
+        }
+      }
+    }
+  }
+
+  .hidden {
+    display: none !important;
+  }
+
+  .extra-shortcuts-list {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    box-sizing: border-box;
+    max-height: calc(100vh - $top-menu-height - $sm-spacing - $filter-height - 2 * $xs-border - $xs-spacing - $sm-spacing - $timeline-tools-height - $timeline-height);
+    overflow-y: auto;
+    position: absolute;
+    top: calc($filter-height + 2 * $xs-border);
+
+    .shortcut {
+      padding-left: 0;
+      margin-top: $xs-margin;
+
+      ::ng-deep.container {
+        align-items: flex-end;
+        width: max-content;
+
+        // Powerbars container
+        ::ng-deep.container {
+          align-items: flex-start;
+          width: 100%;
+        }
+      }
```

Comment:
it seems odd that the same class is used as a child

Reply:
There is the container used in shortcuts, and powerbars have the same class name for their container of all powerbar components.